miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniAppSdkConfig.kt

```diff
@@ -15,7 +17,8 @@ data class MiniAppSdkConfig(
     val subscriptionKey: String,
     val hostAppVersionId: String = "",
     val hostAppUserAgentInfo: String,
-    val isPreviewMode: Boolean
+    val isPreviewMode: Boolean,
+    val miniAppAnalyticsConfigs: List<MiniAppAnalyticsConfig> = emptyList()
```

Comment:
`miniAppAnalyticsConfigs` looks like a typo. How about this instead?

```suggestion
    val miniAppAnalyticsConfigList: List<MiniAppAnalyticsConfig> = emptyList()
```